flight/net/Request.php

```diff
@@ -147,7 +147,7 @@ public function __construct(array $config = [])
                 'scheme' => self::getScheme(),
                 'user_agent' => self::getVar('HTTP_USER_AGENT'),
                 'type' => self::getVar('CONTENT_TYPE'),
-                'length' => self::getVar('CONTENT_LENGTH', 0),
+                'length' => (int)self::getVar('CONTENT_LENGTH', 0),
```

Comment:
```suggestion
                'length' => (int) self::getVar('CONTENT_LENGTH', 0),
```
Better with space after cast.